packages/core/src/local-admin-api.ts

```diff
@@ -4,22 +4,28 @@ import {
   ModelData,
   NodeStatusResponse,
   PinApi,
+  LoadOpts,
 } from '@ceramicnetwork/common'
-import { StreamID } from '@ceramicnetwork/streamid'
+import { Model } from '@ceramicnetwork/stream-model'
+import { StreamID, CommitID } from '@ceramicnetwork/streamid'
 import { LocalIndexApi } from './indexing/local-index-api.js'
-import { SyncApi } from './sync/sync-api.js'
+import { ISyncApi, ModelSyncOptions } from './sync/interfaces.js'
+import { convertCidToEthHash } from '@ceramicnetwork/anchor-utils'
 
 type NodeStatusFn = () => Promise<NodeStatusResponse>
+type LoadStreamFn<T> = (streamId: StreamID | CommitID | string, opts?: LoadOpts) => Promise<T>
 
+export const NUMBER_OF_BLOCKS_BEFORE_TX_TO_START_SYNC = 360 // >24 hours
```

Comment:
are you sure this is more than 24 hours?  Ethereum blocks are produced once every 12 seconds so by my math 360 blocks is only 1.2 hours?